tests/ti/cc3220_launchpad/common/application_code/main.c

```diff
@@ -383,22 +390,22 @@ void vApplicationGetTimerTaskMemory( StaticTask_t ** ppxTimerTaskTCBBuffer,
 /*-----------------------------------------------------------*/
 
 /**
- * @brief Socket event handler function.
```

Comment:
This is no longer needed. It was deleted in the demo main.c, but forgotten in the test main.c.

Reply:
whoops! good catch :)